tests/core/full_node/test_mempool.py

```diff
@@ -1074,3 +1493,331 @@ async def test_my_amount_too_large(self, two_nodes):
         assert sb1 is None
         assert status == MempoolInclusionStatus.FAILED
         assert err == Err.ASSERT_MY_AMOUNT_FAILED
+
+    @pytest.mark.asyncio
+    async def test_unknown_condition(self, two_nodes):
+
+        full_node_1, full_node_2, server_1, server_2 = two_nodes
+        cvp = ConditionWithArgs(ConditionOpcode.UNKNOWN, [])
+        dic = {cvp.opcode: [cvp]}
+        blocks, spend_bundle1, peer, status, err = await self.condition_tester(two_nodes, dic)
+
+        sb1 = full_node_1.full_node.mempool_manager.get_spendbundle(spend_bundle1.name())
+
+        assert sb1 is None
+        assert status == MempoolInclusionStatus.FAILED
+        assert err == Err.INVALID_CONDITION
+
+    def test_parse_condition_agg_sig(self):
+
+        valid_pubkey = b"b" * 48
```

Comment:
This may not actually be a valid pubkey... does it parse with `G1Element.parse` or whatever? Not sure how much difference it makes, but it might make SOME. (I understand you're just checking length here.)

Reply:
Oh I see, it's barely used anywhere... just one call down into the stack. Okay.